apiserver/pkg/util/config.go

```diff
@@ -13,7 +13,8 @@ const (
 	RayClusterUserLabelKey            = "ray.io/user"
 	RayClusterVersionLabelKey         = "ray.io/version"
 	RayClusterEnvironmentLabelKey     = "ray.io/environment"
-	RayServiceLabelKey                = "ray.io/service"
+	RayOriginatedFromNameLabelKey     = "ray.io/originated-from-name"
+	RayOriginatedFromTypeLabelKey     = "ray.io/originated-from-type"
```

Comment:
`ray.io/owner-resource`?

Reply:
As @kevin85421 mentioned, we prefer not to use the term `owner` as it frequently sets incorrect expectations.